tests/test_benchmark.py

```diff
@@ -87,3 +87,20 @@ def test_run_mpi(mode):
         'dump_norms': 'norms.txt'
     }
     run('acoustic', **kwargs)
+
+
+@skipif('only-advisor')
```

Comment:
since it's called "skipif", instead of calling it "only-advisor" I'd use something else, for example "noadvisor" , which would also be similar to the "nodevice" that we already have in place

Reply:
right!